robocorp-code/vscode-client/src/conversion.ts

```diff
@@ -200,8 +200,16 @@ export async function convertAndSaveResults(
     try {
         let nextBasename: string;
         switch (opts.inputType) {
-            case RPATypes.uipath:
+            case RPATypes.uipath: {
+                const projects: Array<string> = opts.input;
                 nextBasename = await findNextBasenameIn(opts.outputFolder, "converted-uipath");
+                rpaConversionCommands.push({
+                    command: CommandType.Schema,
+                    vendor: Format.UIPATH,
+                    projects: projects,
+                    onProgress: undefined,
+                    outputRelativePath: join(nextBasename, "schema"),
```

Comment:
Question (it seems fine for me, just double-checking): if there are multiple input files, only a single `schema` is actually expected?

Reply:
yes, the schema is the commonality across all bots. It works just like aav11.